pretyping/detyping.ml

```diff
@@ -476,7 +476,7 @@ let rec detype flags avoid env sigma t = CAst.make @@
  	| _ -> GApp (f',args')
       in
       mkapp (detype flags avoid env sigma f)
-        (detype_array flags avoid env sigma args)
+        (Array.map_to_list (detype flags avoid env sigma) args)
```

Comment:
Actually, this restores the standard detyping order, since @ppedrot had made it backward when he hand-coded the `detype_array` function, arguing that `detype` is pure. Now the point is moot.